src/ui/components/AccountSelector.js

```diff
@@ -29,22 +27,45 @@ export default function AccountDropdown({
   ledger,
 }: DropdownProps) {
   const items = ledger.accounts().filter((a) => a.active);
+  const [selectedAccount, setSelectedAccount] = useState<Account | null>(null);
 
-  const onComboChange = (e, inputObj) => setCurrentIdentity(inputObj);
+  const onComboChange = (e, inputObj) => {
+    setSelectedAccount(inputObj);
+    setCurrentIdentity(inputObj);
```

Comment:
similar question as before--why are we calling setCurrentIdentity and setSelectedAccount? the account includes the identity

Reply:
This one is actually necessary as far as I can tell in order to update the balances in the child, since using the setCurrentIdentity only sets the identity at the parent level, and doesn't update the UI for the child level.

Reply:
I will make a second pass today to see if there is a better way to do this. Because I agree if does seem strange.

Reply:
Managed to get rid of this by removing balance displays from the Autocomplete component